### PR TITLE
Remove the time bit, fix headings

### PR DESCRIPTION
**Pull request by ektravel**
### Description

This PR removes the bit about how special the `__time` column is.
Additionally, it fixes the headings making them sentence case.

This PR has:
- [ X ] been self-reviewed.

### CODE REVIEW

File: docs/design/segments.md [hunk 1]
```diff
@@ -193,9 +193,7 @@ A segment contains several files:
 
 * `XXXXX.smoosh`
 
-    A number of files containing concatenated binary data.
-
-    The `smoosh` files represent multiple files "smooshed" together in order to minimize the number of file descriptors that must be open to house the data. They are files of up to 2GB in size (to match the limit of a memory mapped ByteBuffer in Java). The `smoosh` files house individual files for each of the columns in the data as well as an `index.drd` file with extra metadata about the segment.
+    Smoosh (`.smoosh`) files contain concatenated binary data. This file consolidation reduces the number of file descriptors that must be open when accessing data. The files should be 2 GB or less in size to remain within the limit of a memory mapped `ByteBuffer` in Java. Smoosh files contain individual files for each column in the data and an `index.drd` file that contains additional segment metadata.
```
Comment (vtlim) [inline docs/design/segments.md:196]:
```suggestion
    Smoosh (`.smoosh`) files contain concatenated binary data. This file consolidation reduces the number of file descriptors that must be open when accessing data. The files are 2 GB or less in size to remain within the limit of a memory-mapped `ByteBuffer` in Java. Smoosh files contain individual files for each column in the data and an `index.drd` file that contains additional segment metadata.
```

Comment (vtlim) [reply docs/design/segments.md:196]: Another suggestion, to integrate the `__time` column:
```suggestion
Smoosh (`.smoosh`) files contain concatenated binary data. This file consolidation reduces the number of file descriptors that must be open when accessing data. The files are 2 GB or less in size to remain within the limit of a memory-mapped `ByteBuffer` in Java. Smoosh files contain the following:
* Individual files for each column in the data, including one for the `__time` column that refers to the time column of the segment
* An `index.drd` file that contains additional segment metadata
```

Comment (techdocsmith) [reply docs/design/segments.md:196]: +1 for Victoria's suggestion

**Review by techdocsmith — APPROVED** [review]
LGTM after @vtlim 's suggestions

File: docs/design/segments.md (hunk 1 above)
Comment (ektravel) [reply docs/design/segments.md:196]: I think we should make it explicit that the `__time` column refers to the timestamp of the segment.

**Review by vtlim — APPROVED** [review]
LGTM 🦖

**Pull request merged**